upi/gcp/template/06_worker.py

```diff
@@ -0,0 +1,71 @@
+def GenerateConfig(context):
+
+    resources = [{
+        'name': context.properties['infra_id'] + '-worker-0',
+        'type': 'compute.v1.instance',
+        'properties': {
+            'disks': [{
+                'autoDelete': True,
+                'boot': True,
+                'initializeParams': {
+                    'diskSizeGb': context.properties['root_volume_size'],
+                    'sourceImage': context.properties['image_name']
+                }
+            }],
+            'machineType': 'zones/' + context.properties['region'] + '-a/machineTypes/' + context.properties['machine_type'],
+            'metadata': {
+                'items': [{
+                    'key': 'user-data',
+                    'value': context.properties['ignition']
+                }]
+            },
+            'networkInterfaces': [{
+                'subnetwork': context.properties['compute_subnet']
+            }],
+            'serviceAccounts': [{
+                'email': context.properties['service_account_email'],
+                'scopes': ['https://www.googleapis.com/auth/cloud-platform']
+            }],
+            'tags': {
+                'items': [
+                    context.properties['infra_id'] + '-worker',
+                ]
+            },
+            'zone': context.properties['region'] + '-a'
+        }
+    }, {
+        'name': context.properties['infra_id'] + '-worker-1',
```

Comment:
I'm not familiar with how GCP templates stack, but in AWS the template is for a single compute machine, and you can [run it as many times as you like][1] to create multiple machines.  Is that feasible for GCP?  Or will folks have to edit this template if they decide they want more or less than two compute nodes?

[1]: https://github.com/openshift/release/blob/ee0fd2b8c7f348459d1e9b53a303964de3f4f294/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L550-L574

Reply:
We can do that. I've applied it to workers while keeping contrl_plane at 3.